source/manuals/programming-languages/java.html.md.erb

```diff
@@ -91,6 +93,16 @@ var checkResponse = service.getCheckResponse();
 
 Be mindful that `var` hides the type of the variable. If the variable name makes the type obvious, this usually isn’t a problem. But if it is not clear from either the variable name or the right-hand side of the assignment, it might be better to explicitly write the type.
 
+If you use `var` in place of a generic type, this will usually prevent you from using the so-called diamond operator (itself a form of type inference) on the right-hand side of the assignment:
```

Comment:
This seems to be implying that var example below won't work

Reply:
Does it? The intention is to first show a typical use of the diamond operator, then show how if we replace the type with `var` we can no longer use the diamond operator and have to specify the generic type on the right-hand side.

Reply:
it might be clearer perhaps to say that when using var you have to specify the type on the right hand side of the assignment, rather than on the left with a diamond operator on the right

Reply:
```suggestion
If you use `var` in place of a generic type you should specify the generic type inside the diamond operator on the right hand side.  If you don't, the compiler will give you a `Foo<Object>` which you probably don't want (and if you do want that, you should make that clear by using `<Object>` instead of `<>`):
```

Reply:
I’ve reworded it. How’s it read now?